src/events/pollVotationResults.js

```diff
@@ -10,11 +10,15 @@ module.exports = {
       return;
     }
 
+    if (!embeds[0]?.fields) {
+      return;
+    }
```

Comment:
You could move this early return to the line 9:

```
if (author.bot && Array.isArray(embeds[0]?.fields) && embeds[0]?.data?.type !== 'poll_result') {
```

![image](https://github.com/user-attachments/assets/853bf46b-4555-4c2d-bd1d-48549cc2ef5a)

```suggestion
```

Reply:
Completed